src/components/homePage/Testimonials.vue

```diff
@@ -1,13 +1,32 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <h2>Testimonials</h2>
+    <!-- SIDE SCROLLER -->
+    <vueper-slides>
+      <vueper-slide 
+        v-for="(slide, i) in information['slides']" 
+        :key="i" 
+        :title="slide.title" 
+        :content="slide.content" 
+        :image="slide.image" 
+      />
+    </vueper-slides>
+  </div>
 </template>
 
 <script>
+import { VueperSlides, VueperSlide } from 'vueperslides';
+import 'vueperslides/dist/vueperslides.css';
```

Comment:
Is this necessary for our use case?

Reply:
Do you mean the css import?

Reply:
Yes I was referring to the css import, my bad